bench_manager/bench_manager/doctype/bench_settings/bench_settings.py

```diff
@@ -389,6 +410,7 @@ def backup_sites_with_daily_option():
     site_list = frappe.get_list("Site",filters={"frequency":"Daily","auto_backup":1})
     if site_list:
         create_backup(site_list)
+        
```

Comment:
sites with auto bacukup and dropbox backed twice once in this hook and once in the other one